README.md

```diff
@@ -29,6 +29,13 @@ from the project root. This will create a plugin zip file at
 `bazel-bin/ijwb/ijwb_bazel.zip`, which can be installed directly
 from the IDE.
 
+For CLion, replace all mentions of `ijwb` above to `clwb`. If CLion refuses
```

Comment:
We also support Android Studio with `//aswb:aswb_bazel_zip`
We can probably just list the three things.